apps/web/playwright/fixtures/eventTypes.ts

```diff
@@ -0,0 +1,94 @@
+import { expect, type Page } from "@playwright/test";
+
+import { localize } from "../lib/testUtils";
+
+export function createEventTypeFixture(page: Page) {
+  return {
+    goToEventType: async (eventType: string) => {
+      await page.getByRole("link", { name: eventType }).click();
+    },
+    goToTab: async (tabName: string) => {
+      await page.getByTestId(`vertical-tab-${tabName}`).click();
+    },
+    goToEventTypesPage: async () => {
+      await page.goto("/event-types");
+    },
+    checkAvailabilityTab: async () => {
+      const editAvailability = (await localize("en"))("edit_availability");
+
+      // Verify if the icon is rendered
+      await expect(page.locator("span").filter({ hasText: "Europe/London" }).locator("svg")).toBeVisible();
+      await expect(page.getByText("Europe/London")).toBeVisible();
+      await page.getByRole("link", { name: editAvailability }).click();
+    },
+    goToAvailabilityPage: async () => {
+      const workingHours = (await localize("en"))("default_schedule_name");
+
+      await page.goto("/availability");
+      await page
+        .getByTestId("schedules")
+        .locator("div")
+        .filter({
+          hasText: workingHours,
+        })
+        .first()
+        .click();
+    },
+    checkAvailabilityPage: async () => {
+      const sunday = (await localize("en"))("sunday");
+      const monday = (await localize("en"))("monday");
+      const wednesday = (await localize("en"))("wednesday");
+      const saturday = (await localize("en"))("saturday");
+      const save = (await localize("en"))("save");
+      const copyTimesTo = (await localize("en"))("copy_times_to");
+      const availabilityUpdatedSuccessfully = (await localize("en"))("availability_updated_successfully");
+
+      await page.getByTestId("availablity-title").click();
+      // change availability name
+      await page.getByTestId("availablity-title").fill("Working Hours test");
+      await expect(page.getByTestId("subtitle")).toBeVisible();
+      await page.locator("label").filter({ hasText: sunday }).getByRole("switch").click();
+      await page.locator("div").filter({ hasText: monday }).first().click();
+      await page.locator("label").filter({ hasText: wednesday }).getByRole("switch").click();
+      await page.locator("label").filter({ hasText: saturday }).getByRole("switch").click();
+      await page
```

Comment:
Should we not change these to use test IDs so we don’t have to filter by text?

Reply:
makes sense, we provided this changes now